lola.html

```diff
@@ -670,19 +664,92 @@ <h4>Copying “replies” and “inReplyTo” </h4>
       <h4>Adding Breadcrumbs</h4>
       
       <p>
-        As the destination server copies Activity and Like objects, it SHOULD add breadcrumbs.  It SHOULD preserve the existing breadcrumbs because it’s nice to do so.  Breadcrumbs should be pushed on the top (or beginning) of a most-recently-first list in the ‘previously’ attribute.
+        As the destination server copies Activity and Like objects, it SHOULD add breadcrumbs.  It SHOULD preserve the existing breadcrumbs because it’s nice to do so.  Breadcrumbs should be pushed on the top (or beginning) of a most-recently-first list in the ‘previously’ property.  The 'previously' attribute is defined in this specification as a multi-valued property, as follows: 
+      </p>
+
+      <table>
+        <tr><td>URI:</td><td>TBD</td></tr>
+        <tr><td>Notes:</td><td>One or more actor/ID pairs</td></tr>
+        <tr><td>Domain:</td><td>Activity</td></tr>
+        <tr><td>Range:</td><td>Object | Link</td></tr>
+      </table>
+
+      <p>Example:</p>
+      <pre>
+"type": "Article",
+"id": "https://thirdhome.example/meadowvine/posts/xyz",
+"previously": [
+  { 
+    "actor": "https://newsite.example.org/aurora/",
+    "id": "https://newsite.example.org/aurora/items/02751cab-..."
+  },
+  { 
+    "actor": "https://lemongrove.example.co.uk",
+    "id": "https://lemongrove.example.co.uk/2016/05/minimal-activitypub"
+  }
+]
+      </pre>
+      <p>The "actor" value SHOULD be included, because there is no guarantee that a 3rd party can derive the actor value from 
+        an "id" value. The "actor" value is needed in order to validate the previous "id" value before trusting it.  Because
+        the destination server simply states the "previously" value, it cannot be entirely trusted.  A 3rd party MAY check the 
+        original actor to make sure that it does redirect to the new Actor, but that check is not guaranteed to work because the
+        source server hosting the original Actor may be inactive or have removed that Actor.  
```

Comment:
maybe it's not worth spelling out explicitly in the spec, but what should a client do if previously values redirect to... some other URL not in the chain of previouslies? should they drop the whole thing as toxic/fishy or are they conforming with this spec to just delete some or all entries in `previously`? it's the cornerest of corner cases, but maybe worth sticking in the list of things to consider at the end when security considerations get written (maybe not worth thinking about now, as the whole `previously` design could change, leaving this nit in the "overtaken by events" bucket!)